models/public/octave-resnet-200-0.125/octave-resnet-200-0.125.md

```diff
@@ -15,30 +15,30 @@ The `octave-resnet-200-0.125` model is a modification of [`resnet-200`](https://
 
 ## Accuracy
 
-| Metric | Value |
-| ------ | ----- |
-| Top 1  | 79.99%|
+| Metric | Value  |
+| ------ | ------ |
+| Top 1  | 79.99% |
 | Top 5  | 94.866%|
 
 ## Input
 
-A blob that consists of a single image of `1x3x224x224` in `RGB` order. Before passing the image blob into the network, subtract RGB mean values as follows: [124,117,104]. In addition, values must be divided by 0.0167.
+A blob that consists of a single image of `1, 3, 224, 224` in `RGB` order. Before passing the image blob into the network, subtract RGB mean values as follows: [124, 117, 104]. In addition, values must be divided by 0.0167.
```

Comment:
the same comment about moving into description

Reply:
Updated